### PR TITLE
translate Cluster Training and Prediction

### LINKED ISSUE #8954: Translation Plan-集群训练与预测-汉译英
## 任务背景
针对v2版本，目前我们已有很多中文使用文档，但仍有大量的汉译英工作及少量英译汉工作没有完成，我们希望将中英文版本补充完全，提升用户的阅读体验，也有益于国外用户在使用时查阅文档。
此外，我们的模型库Models，仍有大部分英译汉和汉译英的工作，亟待完成。

## 任务描述
1. 需要做什么：
把“集群训练与预测”汉译英，英文标题为Cluster Training and Prediction，请勿改动
2. 地址
    - 中文地址： [Github地址](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/v2/faq/cluster/index_cn.rst)，[网页地址](http://www.paddlepaddle.org/docs/develop/documentation/zh/faq/cluster/index_cn.html)
    - 需要翻译的英文地址： [Github地址](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/v2/faq/cluster/index_en.rst)，[网页地址](http://www.paddlepaddle.org/docs/develop/documentation/en/faq/cluster/index_en.html)

## 如何认领
我们欢迎大家积极认领，内部小伙伴请通过self assign的方式认领，外部小伙伴可通过在本issue下方comment认领。

## 注意事项
上交初稿的截止日期是2018年4月15日
上交后，PaddlePaddle的RD会为大家review
请于review后的5个工作日之内完成修改

## 相关信息
所有需要翻译的文档：
V2使用文档：
https://shimo.im/sheet/PqK407VYm80Ju3dw
Models:
https://shimo.im/sheet/6XTZnRQEUS0uC1Ut

### PR DESCRIPTION
**Pull request by m3ngyang**
fix #8954

### CODE REVIEW

File: doc/v2/faq/cluster/index_en.rst [hunk 1]
```diff
@@ -2,4 +2,15 @@
 Cluster Training and Prediction
 ###############################
 
-TBD
+.. contents::
+
+1. Network connection errors in the log during muliti-node cluster training
+------------------------------------------------
+The errors in the log belong to network connection during mulilti-node cluster training, for example, :code:`Connection reset by peer`.
```
Comment (typhoonzero) [inline doc/v2/faq/cluster/index_en.rst]: This sentence only have subject but no predicate and no object.

File: doc/v2/faq/cluster/index_en.rst [hunk 2]
```diff
@@ -2,4 +2,15 @@
...
+This kind of error is usually caused by the abnormal exit of the training process in some node, and the others cannot connect with this node any longer. Steps to troubleshoot the problem as follows:
+
+* Find the first error in the :code:`train.log`, :code:`server.log`, check whether other fault casued the problem, such as FPE, lacking of memory or disk.
+
+* If network connection gave rise to the first error in the log, this may be caused by the port conflict of the non-exclusive execution. Connect with the operator to check if the current MPI cluster supports jobs submitted with parameter :code:`resource=full`. If so, change the port of job.
```
Comment (typhoonzero) [inline doc/v2/faq/cluster/index_en.rst]: "If network connection gave rise to the first error in the log" => If the first error in `server.log` says "Address already used"

Comment (typhoonzero) [inline doc/v2/faq/cluster/index_en.rst]: Connect with the operator => Contact the sys-admin

Comment (typhoonzero) [inline doc/v2/faq/cluster/index_en.rst]: If so, change the port of job. => If the current MPI cluster does not support this parameter, change the server port and try agian.

File: doc/v2/faq/cluster/index_en.rst [hunk 3]
```diff
@@ -2,4 +2,15 @@
...
+
+* If the currnet MPI cluster does not support exclusive pattern, ask the operator to replace or update the current cluster.
```
Comment (typhoonzero) [inline doc/v2/faq/cluster/index_en.rst]: people may want to know what the "exclusive pattern" is.

Comment (typhoonzero) [reply doc/v2/faq/cluster/index_en.rst]: operator => cluster administrator

Comment (m3ngyang) [reply doc/v2/faq/cluster/index_en.rst]: > people may want to know what the "exclusive pattern" is.
Which doc should we refer this term to ?

File: doc/v2/faq/cluster/index_en.rst [hunk 4]
```diff
@@ -2,4 +2,15 @@
 Cluster Training and Prediction
 ###############################
 
-TBD
+.. contents::
+
+1. Network connection errors in the log during muliti-node cluster training
```
Comment (abhinavarora) [inline doc/v2/faq/cluster/index_en.rst]: muliti-node -> **multi-node**

File: doc/v2/faq/cluster/index_en.rst [hunk 5]
```diff
@@ -2,4 +2,15 @@
...
+------------------------------------------------
```
Comment (abhinavarora) [inline doc/v2/faq/cluster/index_en.rst]: mulilti-node -> **multi-node**

File: doc/v2/faq/cluster/index_en.rst [hunk 6]
```diff
@@ -2,4 +2,15 @@
...
+The errors in the log belong to network connection during mulilti-node cluster training, for example, :code:`Connection reset by peer`.
+This kind of error is usually caused by the abnormal exit of the training process in some node, and the others cannot connect with this node any longer. Steps to troubleshoot the problem as follows:
```
Comment (abhinavarora) [inline doc/v2/faq/cluster/index_en.rst]: of the training process -> of **a** training process

Comment (abhinavarora) [reply doc/v2/faq/cluster/index_en.rst]: others -> **other nodes**

Comment (abhinavarora) [reply doc/v2/faq/cluster/index_en.rst]: the problem as follows -> the problem **are** as follows

File: doc/v2/faq/cluster/index_en.rst (hunk 3 above)
Comment (abhinavarora) [inline doc/v2/faq/cluster/index_en.rst]: currnet -> **current**

**Comment by m3ngyang** [comment]
done

**Pull request merged**